pkg/shell/runner.go

```diff
@@ -65,14 +74,30 @@ func (r *shellRunner) executeCmd(obfuscate func([]string), cmdExe string, cmdArg
 		obfuscate(displayValues)
 	}
 	logrus.Infof("Running command %s %s", cmdExe, strings.Join(displayValues, " "))
-	return r.execute(cmd)
+	var output io.Writer
```

Comment:
I think if you change readOuputs from bool to io.Writer, and QueryCmd pass *bytes.Buffer and ExecuteCmd pass nil, the code will be cleaner.

Reply:
sure